drivers/VDI.py

```diff
@@ -519,9 +522,12 @@ def _db_update_sm_config(self, ref, sm_config):
                 self.sr.session.xenapi.VDI.add_to_sm_config(ref, key, val)
 
         for key in current_sm_config.keys():
-            if key.startswith("host_") or \
-                key in ["paused", cleanup.VDI.DB_VHD_BLOCKS] or \
-                key in self.sm_config_keep:
+            if (key.startswith("host_") or
```

Comment:
we used to have backslashes here, and now we don't anymore, I assume that is fine?

Reply:
I added () which is a nicer way of keeping the boolean contained.

Reply:
But, I did forget that I said I would move the list of keys to a single constant, so will update with that